package.json

```diff
@@ -10,25 +8,39 @@
     "scss",
     "web component"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BrightspaceUI/colors.git"
+  },
+  "name": "d2l-colors",
   "scripts": {
-    "postinstall": "polymer install --variants",
     "build:sass": "node-sass ./test/test.scss ./test/test.css",
-    "test:lint": "npm run test:lint:wc && npm run test:lint:html",
-    "test:lint:html": "eslint *.html",
+    "test:lint": "npm run test:lint:wc && npm run test:lint:js",
+    "test:lint:js": "eslint . test/** demo/** --ext .js,.html",
```

Comment:
This seems to result in linting any `.css` and `.scss` files in test directory, which of course fails because eslint only handles JS.